.pre-commit-config.yaml

```diff
@@ -29,13 +29,13 @@ repos:
   - id: mixed-line-ending
   - id: requirements-txt-fixer
   - id: trailing-whitespace
-  - id: fix-encoding-pragma
     exclude: ^noxfile.py$
```

Comment:
```suggestion
```

Reply:
^ This is a hook specific include that should be removed along with the offending hook.